_posts/blog/2018-09-18/2018-09-18-5 -tips-to-make-your-openbazaar-listings-pop.markdown

```diff
@@ -0,0 +1,158 @@
+
+Absolutely irresistible.
+
+### 4. Break up long blocks of text
+
+Product descriptions or store policies can get lengthy. Don’t make your buyer slog through line after line of text. Instead, format your content to help them find the information they need.
+
+#### Write short paragraphs
+
+Keep your paragraphs to 3-5 lines max. If you have a lot to communicate, try organizing the content into sections with header titles. It will make the information much less intimidating and easier to digest. For example, in your product description, you can have sections for _key features_, _dimensions_, or _care instructions_.
+
+#### Use bullet points
+
+Bullet points are your friends! They’re easy to read and convey a lot of information very quickly. Use them when you can. They’re especially helpful when listing product features.
+
+#### Format your text
+
+A little text formatting can go a long way. Call out important tidbits in **bold** or _italics_ so they don’t get lost.
+
+#### Add visuals 
+
+Get creative! Break up your text with photos and GIFs. Visuals can add an instant POP to your content, making it dynamic and engaging. For example, check out [this listing by Crypto Republic - BCH](https://openbazaar.com/store/QmU5ZSKVz2GhsqE6EmBGVCtrui4YhUXny6rbvsSf5h2xvH/mini-metro-cd-key-steam).
+
+![Creating Listing Tags on OpenBazaar](Creating Listing Tags on OpenBazaar.gif "Creating Listing Tags on OpenBazaar")
+
+### 5. Add [tags](https://openbazaar.org/blog/how-to-get-more-buyers-on-openbazaar-with-categories-and-tags/) to get your listing discovered!
+
+Finally, make sure you tag your products so people can find them on the network. You’ve put a lot of blood and sweat into creating this listing, so you deserve to get some eyeballs on it. 
+
+When tagging your listing, try to think like a shopper – if you were looking for a product like this, what kind of search terms would you use? Some questions you can ask yourself include:
+
+#### How would a shopper describe this item?
+
+For example, if I were looking for a pair of green earrings, I might search:
+
+- _Jewelery_
+- _Earrings_
+- _Studs_
+- _Green_
+- _Emerald_
+
+#### Why would a shopper be looking for this item?
+
+As a shopper, I might not know exactly what I’m looking for. I could be shopping for a gift for my mom. In that case, I might search:
+
+- _Gifts for her_
+- _Mother’s day_
+
+For more information about tagging, check out our blog post [here](https://openbazaar.org/blog/how-to-get-more-buyers-on-openbazaar-with-categories-and-tags/). 
+
+### Conclusion
+
+Don't get lost in the ecommerce crowd. Make sure your OpenBazaar listings are optimized with great titles, photos, details, formatting and tags so that shoppers are so delighted to discover them they can't help but buy.
+
+Once your listings are all set up, make sure to find your store link on the web at [openbazaar.com](https://openbazaar.com) so you can share them with your audience on social media!
```

Comment:
I think it would be good to mention here that it can take a while for new stores and listings to show up on the .com site.